diff --git a/crates/iota-indexer/src/schema.rs b/crates/iota-indexer/src/schema.rs
--- a/crates/iota-indexer/src/schema.rs
+++ b/crates/iota-indexer/src/schema.rs
@@ -1,4 +1,7 @@
-// @generated automatically by Diesel CLI.
+// Copyright (c) Mysten Labs, Inc.
+// Modifications Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+// @generated automatically by scripts/indexer-schema/generate.sh

 diesel::table! {
     active_addresses (address) {
